src/components/Home/WorksList/WorksList.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Project from "../../ProjectCard/ProjectCard";
+import data from "../../../data.json";
+
+export default function WorksList() {
+    return (
+        <div className="works-list-container">
+            <h2>Works</h2>
```

Comment:
this sounds wrong .. the  word work is like fish .. the plural can be works sometimes .. but has different meaning ... I would either go with **work** or better **Projects**